app/assets/javascripts/discourse/tests/unit/lib/highlight-syntax-test.js

```diff
@@ -0,0 +1,60 @@
+import highlightSyntax from "discourse/lib/highlight-syntax";
+import { module, test } from "qunit";
+import { fixture } from "discourse/tests/helpers/qunit-helpers";
+
+let siteSettings = { autohighlight_all_code: true },
+  session = {
+    highlightJsPath: "/assets/highlightjs/highlight-test-bundle.min.js",
```

Comment:
Curious, why is it that we can't use the regular (non-test) bundle?

Reply:
Only thing I saw differently is that it's a smaller bundle, but I'm not sure why we created this thing in the first place. Maybe it was the "run tests from themes in the running app" initiative? I don't have enough context here.

Reply:
Oh, I see, it was already there... ok, nevermind.